setup.py

```diff
@@ -52,7 +52,12 @@ def read(fname):
 requires = []
 for dep in info.get('depends', []):
     if not re.match(r'(ir|res|webdav)(\W|$)', dep):
-        requires.append('trytond_%s >= %s.%s, < %s.%s' %
+        if dep == 'product_notebook':
```

Comment:
why are you doing this?? Just make product-notebook dependency of nereid-catalog.